csrc/executor_utils.cpp

```diff
@@ -832,6 +832,10 @@ void bindInputForExprEvaluation(
     ExpressionEvaluator& expr_eval) {
   if (val->getValType() == ValType::TensorView) {
     TensorView* cg_tensor = val->as<TensorView>();
+    auto tensor_arg_abstract = dynamic_cast<const TensorArgAbstract*>(arg);
+    if (tensor_arg_abstract != nullptr) {
+      expr_eval.bind(cg_tensor, tensor_arg_abstract->getTensor());
```

Comment:
Here, we are binding the metadata of `tensor_arg_abstract`?

Reply:
We are binding `tensor_arg_abstract->getTensor()`, which is an `at::Tensor`.

Reply:
I guess what I'm wondering is when an aten tensor is bound to a TensorView, are we binding its metadata as well? If so, can we remove the code below this part?

Reply:
Yes, when binding an aten tensor, we should have all its metadata. The code below should be removed, but needs more cleanup (which will be done in my future PR) in order to do so. For example, tensor sizes are currently still named scalar `T1.size[0]`.